content/millennium/r4/diagnostic/observation.md

```diff
@@ -191,7 +191,7 @@ _Implementation Notes_
 
```

Comment:
Need to call out that Vitals creates for a Provider persona requires an active relationship

Reply:
<img width="615" alt="Screen Shot 2021-06-17 at 2 47 12 PM" src="https://user-images.githubusercontent.com/18075594/122463228-f77c2480-cf7a-11eb-81b9-b2da2e07dffb.png">

Reply:
nit: `Vital Sign creates via a Provider persona requires an active relationship between the Provider and the Patient`

Reply:
<img width="709" alt="Screen Shot 2021-06-17 at 4 25 17 PM" src="https://user-images.githubusercontent.com/18075594/122474309-aa06b400-cf88-11eb-96c5-ce2257174d94.png">